micro-rdk/src/common/webrtc/grpc.rs

```diff
@@ -93,7 +93,7 @@ where
             stream: None,
             headers: None,
             streams: HashMap::new(),
-            buffer: BytesMut::zeroed(1650),
+            buffer: BytesMut::zeroed(15000),
```

Comment:
@npmenard I used the Python SDK with `disable_webrtc=True` but the native server still went down this path with webrtc. I need to try again with the Go SDK, but this change did allow me to get everything working.

Not sure what this value should be if it's the limiting factor. We could feature-gate whatever the value is as well.